src/activities/menu/ConfigurationItem.qml

```diff
@@ -404,6 +413,58 @@ Item {
                 }
             }
         }
+        
+        Flow {
```

Comment:
If I'm not wrong for wordset, the behavior is to check the box in config and then we use the complete wordset (note aside, it seems buggy for now as the only moment where we can actually download the rcc is at start up and if we check the box and don't have dataset we will have error and going to the activities, so for me we should ask to download the rcc if not present and if the user does not want, we keep the embedded wordset).
What you propose would be to download them (or ask user if he wants to download) when we check the "Enable background music" if they are not present (and we can probably go further as not showing it if we are not connected to network)?